interfaces/builtin/common.go

```diff
@@ -142,3 +145,15 @@ func (iface *commonInterface) SecCompConnectedPlug(spec *seccomp.Specification,
 	}
 	return nil
 }
+
+func (iface *commonInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.Plug, plugAttrs map[string]interface{}, slot *interfaces.Slot, slotAttrs map[string]interface{}) error {
+	old := "###SLOT_SECURITY_TAGS###"
```

Comment:
I used format first but then changed this so that it is consistent with `###replaceme###` markers we use elsewhere in the interface code. We can revisit that but then we should do it consistently in the whole stack (all of `builtin/*.go`)